python_bindings/tests/end-to-python-end-test/all_tiles_test.py

```diff
@@ -0,0 +1,1460 @@
+           ..|.........|...\ /..\ /..\ /.
+2          --6----8----A....C...|D|..|E|.
+           ..|.../ \../ \../ \../ \../ \.
+           ..|..|   || @ ||   || ! || @ |
+           ..|..|   |                   |
+           ..|...\ /        *          /.
+1          --5....F !  O    L    M    G..
+           ..|.....\       /.\  /|\  /...
+           ..|......-     -...--.|.--..@.
+      -......|.......|   |.....
+       \.....|........\ /..[@].
+0    *  H....4----3----0---[2].
+       ! \.................[ ].
+          |..@.................
+                     .........--...!
+                     .[!]..../  \... 
+-1                   .[1]...7    9--     
+                     .[ ]....\  /|..
+                     .........--.|..
+               
+"""
+
+
+def check_turn_O(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=0, y=1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NoneMeeple,
+        side=Side.NoSide,
+        featureType=FeatureType.NoneType,
+    )
+
+    game_id, game = make_turn(
+        engine, game, game_id, turn_params, True, {1: 11 + 34 + 11, 2: 8 + 34 + 22}
+    )
+    check_points(game, [11 + 34, 8 + 34])
+
+    return game_id, game
+
+
+"""
+player1 scores 11 points in total:
+    - 1+1 points for a city with shield on H tile
+    - 5 points for monastery on 1 tile
+    - 3 points for farmer on 9 tile    
+    - 0 points for farmer on B tile
+    - 1 point for road on J tile
+    
+player2 scores 22 points in total:   
+    - 3*3points for farmer on 4 tile
+    - 8 points for monastery on 2 tile
+    - 3 points for farmer on G tile
+    - 2 points for road on I tile 
+
+       -3   -2   -1    0    1    2    3
+
+
+                     -.|.-
+                      \|/ 
+4                    * N    
+
+
+                     |   |..@.--...--.|..    
+                     .\ /...|/  \./  \|..  
+3                    --B----I    K    J-! 
+                     ..|.../         * \.
+                     !.|..|             |
+           ..|.........|..|   ||   || * |
+           ..|.........|...\ /..\ /..\ /.
+2          --6----8----A....C...|D|..|E|.
+           ..|.../ \../ \../ \../ \../ \.
+           ..|..|   ||   ||   ||   ||   |
+           ..|..|   |                   |
+           ..|...\ /        *          /.
+1          --5....F    O    L    M    G..
+           ..|.....\       /.\  /|\  /...
+           ..|......-     -...--.|.--..@.
+      -......|.......|   |.....
+       \.....|........\ /..[@].
+0    *  H....4----3----0---[2].
+       ! \.................[ ].
+          |..@.................
+                     .........--...!
+                     .[!]..../  \... 
+-1                   .[1]...7    9--     
+                     .[ ]....\  /|..
+                     .........--.|..
+"""
+
+
+def finalize(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    check_final_points([9 + 17, 36 + 19])
+    return
```

Comment:
Four problems:
- this function is never called (only defined)
- it says it returns (int, SerializedGame), but actually returns None
- it calls check_final_points with a list argument, but check_final_points expects a dict (lists don't even have .items() method)
- the scores in the argument are completely unrelated to the scores in the previous function

Reply:
removed (became obsolete, when we removed finalize request)